src/mainboard/protectli/vault_adl_p/gpio.c

```diff
@@ -0,0 +1,631 @@
+	/* GPP_CPU_RSVD_22 - GPP_CPU_RSVD_22 */
+	PAD_CFG_NF(GPP_CPU_RSVD_22, NONE, DEEP, NF1),
+	/* GPP_CPU_RSVD_23 - GPP_CPU_RSVD_23 */
+	PAD_CFG_NF(GPP_CPU_RSVD_23, NONE, DEEP, NF1),
+	/* GPP_CPU_RSVD_24 - GPP_CPU_RSVD_24 */
+	PAD_CFG_NF(GPP_CPU_RSVD_24, NONE, DEEP, NF1),
+
+	/* ------- GPIO Group vGPIO ------- */
+
+	/* GPP_VGPIO_0 - GPIO */
+	PAD_CFG_GPO(GPP_VGPIO_0, 0, DEEP),
+	/* GPP_VGPIO_4 - GPIO */
+	PAD_CFG_GPI_TRIG_OWN(GPP_VGPIO_4, NONE, DEEP, OFF, ACPI),
+	/* GPP_VGPIO_5 - GPIO */
+	PAD_CFG_GPIO_BIDIRECT(GPP_VGPIO_5, 1, NONE, DEEP, LEVEL, ACPI),
+	/* GPP_VGPIO_6 - GPP_VGPIO_6 */
+	PAD_CFG_NF(GPP_VGPIO_6, NONE, DEEP, NF1),
+	/* GPP_VGPIO_7 - GPP_VGPIO_7 */
+	PAD_CFG_NF(GPP_VGPIO_7, NONE, DEEP, NF1),
+	/* GPP_VGPIO_8 - GPP_VGPIO_8 */
+	PAD_CFG_NF(GPP_VGPIO_8, NONE, DEEP, NF1),
+	/* GPP_VGPIO_9 - GPP_VGPIO_9 */
+	PAD_CFG_NF(GPP_VGPIO_9, NONE, DEEP, NF1),
+	/* GPP_VGPIO_10 - GPP_VGPIO_10 */
+	PAD_CFG_NF(GPP_VGPIO_10, NONE, DEEP, NF1),
+	/* GPP_VGPIO_11 - GPP_VGPIO_11 */
+	PAD_CFG_NF(GPP_VGPIO_11, NONE, DEEP, NF1),
+	/* GPP_VGPIO_12 - GPP_VGPIO_12 */
+	PAD_CFG_NF(GPP_VGPIO_12, NONE, DEEP, NF1),
+	/* GPP_VGPIO_13 - GPP_VGPIO_13 */
+	PAD_CFG_NF(GPP_VGPIO_13, NONE, DEEP, NF1),
+	/* GPP_VGPIO_18 - GPP_VGPIO_18 */
+	PAD_CFG_NF(GPP_VGPIO_18, NONE, DEEP, NF1),
+	/* GPP_VGPIO_19 - GPP_VGPIO_19 */
+	PAD_CFG_NF(GPP_VGPIO_19, NONE, DEEP, NF1),
+	/* GPP_VGPIO_20 - GPP_VGPIO_20 */
+	PAD_CFG_NF(GPP_VGPIO_20, NONE, DEEP, NF1),
+	/* GPP_VGPIO_21 - GPP_VGPIO_21 */
+	PAD_CFG_NF(GPP_VGPIO_21, NONE, DEEP, NF1),
+	/* GPP_VGPIO_22 - GPP_VGPIO_22 */
+	PAD_CFG_NF(GPP_VGPIO_22, NONE, DEEP, NF1),
+	/* GPP_VGPIO_23 - GPP_VGPIO_23 */
+	PAD_CFG_NF(GPP_VGPIO_23, NONE, DEEP, NF1),
+	/* GPP_VGPIO_24 - GPP_VGPIO_24 */
+	PAD_CFG_NF(GPP_VGPIO_24, NONE, DEEP, NF1),
+	/* GPP_VGPIO_25 - GPP_VGPIO_25 */
+	PAD_CFG_NF(GPP_VGPIO_25, NONE, DEEP, NF1),
+	/* GPP_VGPIO_30 - GPP_VGPIO_30 */
+	PAD_CFG_NF(GPP_VGPIO_30, NONE, DEEP, NF1),
+	/* GPP_VGPIO_31 - GPP_VGPIO_31 */
+	PAD_CFG_NF(GPP_VGPIO_31, NONE, DEEP, NF1),
+	/* GPP_VGPIO_32 - GPP_VGPIO_32 */
+	PAD_CFG_NF(GPP_VGPIO_32, NONE, DEEP, NF1),
+	/* GPP_VGPIO_33 - GPP_VGPIO_33 */
+	PAD_CFG_NF(GPP_VGPIO_33, NONE, DEEP, NF1),
+	/* GPP_VGPIO_34 - GPP_VGPIO_34 */
+	PAD_CFG_NF(GPP_VGPIO_34, NONE, DEEP, NF1),
+	/* GPP_VGPIO_35 - GPP_VGPIO_35 */
+	PAD_CFG_NF(GPP_VGPIO_35, NONE, DEEP, NF1),
+	/* GPP_VGPIO_36 - GPP_VGPIO_36 */
+	PAD_CFG_NF(GPP_VGPIO_36, NONE, DEEP, NF1),
+	/* GPP_VGPIO_37 - GPP_VGPIO_37 */
+	PAD_CFG_NF(GPP_VGPIO_37, NONE, DEEP, NF1),
+
+	/* ------- GPIO Community 2 ------- */
+
+	/* ------- GPIO Group GPP_GPD ------- */
+
+	/* GPD0 - BATLOW# */
+	PAD_CFG_NF(GPD0, UP_20K, PWROK, NF1),
+	/* GPD1 - ACPRESENT */
+	PAD_CFG_NF(GPD1, NATIVE, PWROK, NF1),
+	/* GPD2 - GPIO */
+	PAD_NC(GPD2, NONE),
+	/* GPD3 - PWRBTN# */
+	PAD_CFG_NF(GPD3, UP_20K, PWROK, NF1),
+	/* GPD4 - SLP_S3# */
+	PAD_CFG_NF(GPD4, NONE, PWROK, NF1),
+	/* GPD5 - SLP_S4# */
+	PAD_CFG_NF(GPD5, NONE, PWROK, NF1),
+	/* GPD6 - SLP_A# */
+	PAD_CFG_NF(GPD6, NONE, PWROK, NF1),
```

Comment:
Are you sure it is connected?

Reply:
`SLP_*` signals are used to power off certain power planes on a system depending on current power state the system entered. No point in disabling SLP signals in general

Reply:
Ok, I guess that is common for all mainboards and should be obvious then